style.css

```diff
@@ -1,425 +1,894 @@
-  width: 100%;
-  justify-content: center;
-  background-color: #fff;
-}
-
-#lang {
-  font-family: 'Poppins-normal', sans-serif;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  color: #000;
-  margin-bottom: 0;
-  margin-top: 0;
-  margin-left: 36px;
-  margin-right: 64px;
-  display: flex;
-  justify-content: space-between;
-}
-
-.coding-languages-images {
-  list-style: none;
-  padding: 0;
-  display: flex;
-  flex-direction: column;
-  margin-top: 36px;
-  margin-left: 36px;
-}
-
-.framework {
-  font-family: 'Poppins-normal', sans-serif;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-left: 36px;
-  margin-right: 66px;
-}
-
-.horizontal-line {
-  height: 1px;
-  width: 100%;
-  margin-top: 15px;
-  align-self: center;
-  background: #dfe1e6;
-}
-
-.skill {
-  font-family: 'Poppins-normal', sans-serif;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-left: 36px;
-  margin-right: 66px;
-  margin-top: 24px;
-}
-
-.contact-me {
-  display: grid;
-  padding-top: 50px;
-  padding-bottom: 100px;
-  align-items: center;
-  justify-content: center;
-  background: #e5e5e5;
-}
-
-#contact-div {
-  display: flex;
-  justify-content: center;
-  background-image: url(images/Contact\ me.png);
-  background-size: cover;
-  padding: 25px;
-  padding-top: 35px;
-  padding-bottom: 35px;
-  position: relative;
-}
-
-#contact-line {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  color: #fff;
-  margin-top: 70px;
-  margin-bottom: 0;
-}
-
-#ellipse-3 {
-  position: absolute;
-  right: 0;
-  top: 40px;
-}
-
-#ellipse-5 {
-  position: absolute;
-  bottom: 35px;
-  right: 75px;
-}
-
-#ellipse-12 {
-  position: absolute;
-  top: 60px;
-  left: 150px;
-}
-
-#rectangle-4 {
-  position: absolute;
-  right: 0;
-  top: 15px;
-}
-
-#rectangle-7 {
-  position: absolute;
-  bottom: 35px;
-  right: 35px;
-}
-
-#contact-para {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  text-align: center;
-  color: #ebebff;
-}
-
-form {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-}
-
-#input {
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 300px;
-  border-radius: 8px;
-  padding: 10px;
-  margin: 8px;
-}
-
-textarea {
-  resize: none;
-}
-
-#get-in-touch {
-  width: 130px;
-  height: 45px;
+@media (min-width: 768px) {
+  .container {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .menu {
+    display: flex;
+    border: 8px;
+    margin: 10px;
+    padding: 8px;
+    align-items: center;
+    width: 40px;

```

Comment:
- [ ] Make sure when we use a mobile-first approach, screen size is designed based on mobile-first, so the media query we use `min-width` :100: , but specify for which screen size. If you design for a desktop version, please the media query range should be 768px to 1440px.

Example

- Large mobile screen

```
@media only screen and (min-width: 500px) and (max-width: 767px) {
  .headline-content p {
    max-width: 60%;
  }

```
- Medium size screen

```
@media only screen and (min-width: 768px) and (max-width: 999px) {
  .headline-content h1 {
    font-size: 3.625rem;
  }

```
- Desktop and large screen size

```
@media only screen and (min-width: 1000px) {
  body {
    background-position: -8rem 4rem;
    background-image: url("image/desktop-version-icons/circle\ -1-header.png");
    background-repeat: no-repeat;
  }

```